xbmc/interfaces/legacy/Doxyfile.in

```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ./generated
+OUTPUT_DIRECTORY       = @legacy_doxy_output_dir@
```

Comment:
@notspiff: is it safe to move the Doxyfile? Or better copy it to not break the other buildsystems?